samples/README.md

```diff
@@ -1,51 +1,39 @@
-# Android SDK Samples
+#### Responsible Team: Team Messenger
+
+# Intercom Android SDK Samples
 
 ## Intercom Sample
 
 ### Setting Up
-This is a sample application to demonstrate how to integrate Intercom in a simple app.
-You will need to add your API key and app ID to the SampleApplication
-If you have enabled secure mode you will need to provide HMAC and data
+This is set of sample applications which show how to integrate Intercom in a simple app. To try a sample you will need to add your API key and app ID to the `SampleApplication` class in that sample. If you have enabled Identity Verification you will need to provide a HMAC for your user too.
```

Comment:
`App ID`....uppercase A

Reply:
Might be worth linking https://docs.intercom.com/configure-intercom-for-your-product-or-site/staying-secure/enable-identity-verification-on-intercom-for-android behind `Identity Verification` text

Reply:
Keeping this as it's more consistent with our other Android documentation.